extra-devel/openixcard/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=openixcard
+PKGSEC=devel
+PKGDES="Open Source Implementation of Allwinner/sunxi's Image Manipulation Tool PhoenixCard"
+
+BUILDDEP="cmake"
```

Comment:
You won't need to define this. Our guideline specifies that there is no need to specify software shipped with BuildKit as BUILDDEP.